File: 250722/특정 수 맞히기/catching-specific-number.cpp
#include <iostream>
using namespace std;

int main() {
    int guess = 0, twenty_five = 25;
    while (guess != twenty_five) {
        cin >> guess;
        if (guess < twenty_five) {
            cout << "Higher" << endl;
        } else if (guess > twenty_five) {
            cout << "Lower" << endl;
        } else {
            cout << "Good" << endl;
            break;
        }
    }
    return 0;
}

File: 250723/특정 수 맞히기/catching-specific-number.kt
import java.io.BufferedReader
import java.io.BufferedWriter

fun main() {
    BufferedReader(System.`in`.reader()).use { br ->
        BufferedWriter(System.out.writer()).use { bw ->
            val result = mutableListOf<String>()
            while (true) {
                val num = br.readLine().toInt()
                val item = when {
                    num < 25 -> "Higher"
                    num > 25 -> "Lower"
                    else -> "Good"
                }
                result.add(item)
                if (num == 25) {
                    break
                }
            }
            bw.write(result.joinToString("\n"))
        }
    }
}